rac_aspace/serializers.py

```diff
@@ -1,60 +1,61 @@
 import argparse
-from configparser import ConfigParser
-# import csv
-import os
+import csv
 
 
-config = ConfigParser()
-config.read("local_settings.cfg")
+class Serializer:
 
-file_path = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), config.get(
-        "Destinations", "filename")
-)
+    def __init__(self, filename, delimiter, data):
+        self.filename = filename
+        self.fieldnames = list(data.keys())
+        self.data = data
+        self.delimiter = delimiter
 
-FILE_TYPE_CHOICES = ["csv", "tsv"]
+    def run(self):
```

Comment:
Consider renaming this method to `write_data()` or something more descriptive of what it's actually doing. And see comment above about passing `data` in here.